enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1273 @@
+material changes will be paused.
+
+#### Disabled Strategy
+This strategy indicates that no change management strategy is being enforced by the resource. It always implies that
+the enforcement state at the resource level is unpaused / permissive. This does not always
+mean that material changes are permitted due to change management hierarchies. For example, a MachineConfigPool
+with `strategy: Disabled` would still be subject to a `strategy: MaintenanceSchedule` in the ClusterVersion resource.
+The impact of hierarchy should always be made clear in the change management status of the MachineConfigPool.
+
+#### Assisted Strategy - MachineConfigPool
+Minimally, this strategy will be supported by MachineConfigPool. If and when the strategy is supported by other
+change management capable resources, the configuration schema for the policy may differ as the details of
+what constitutes and informs change varies between resources. 
+
+This strategy is motivated by the desire to support the separation of control-plane and worker-node updates both
+conceptually for users and in real technical terms. One way to do this for users who do not benefit from the
+`MaintenanceSchedule` strategy is to ask them to initiate, pause, and resume the rollout of material
+changes to their worker nodes. Contrast this with the fully self-managed state today, where worker-nodes
+(normally) begin to be updated automatically and directly after the control-plane update (subject to constraints
+like `maxUnavailable` in each MachineConfigPool).
+
+Clearly, if this was the only mode of updating worker-nodes, we could never successfully disentangle the
+concepts of control-plane vs worker-node updates in Standalone environments since one implies the other.
+
+In short (details will follow in the implementation section), the assisted strategy allows users to specify the
+exact rendered [`desiredConfig` the MachineConfigPool](https://github.com/openshift/machine-config-operator/blob/5112d4f8e562a2b072106f0336aeab451341d7dc/docs/MachineConfigDaemon.md#coordinating-updates) should be advertising to the MachineConfigDaemon on
+nodes it is associated with. Like the `MaintenanceSchedule` strategy, it also respects the `pausedUntil`
+field.
+
+#### Manual Strategy - MachineConfigPool
+Minimally, this strategy will be supported by MachineConfigPool. If and when the strategy is supported by other
+change management capable resources, the configuration schema for the policy may differ as the details of
+what constitutes and informs change varies between resources. 
+
+Like the Assisted strategy, this strategy is implemented to support the conceptual and technical separation
+of control-plane and worker-nodes. The MachineConfigPool Manual strategy allows users to explicitly specify
+their `desiredConfig` to be used for ignition of new and rebooting nodes. While the Manual strategy is enabled,

```

Comment:
Is the expectation that if the user chooses to do this, that they would be responsible for selecting the correct rendered config?

Reply:
That is the expectation, with the caveat that "correct" is up to the user as long as it is a rendered and appropriate MachineConfig for the MCP.

Reply:
That is the expectation, with the caveat that "correct" is up to the user as long as it is a rendered and appropriate MachineConfig for the MCP.